Lombiq.Tests.UI.Samples/Tests/JavaScriptTests.cs

```diff
@@ -1,67 +1,50 @@
 using Lombiq.Tests.UI.Extensions;
-using Lombiq.Tests.UI.Services.GitHub;
 using System.IO;
 using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Lombiq.Tests.UI.Samples.Tests;
 
-// Let's suppose you want to write UI tests in Javascript. Why would you want to do that? Unlikely if you are an Orchard
+// Let's suppose you want to write UI tests in JavaScript. Why would you want to do that? Unlikely if you are an Orchard
 // Core developer, but what if the person responsible for writing the tests is not? In the previous training section we
 // discussed using a separate frontend server, with mention of technologies using Node.js. In that case the frontend
-// developers may be more familiar with Javascript so it makes sense to write and debug the tests in Node.js so they
+// developers may be more familiar with JavaScript so it makes sense to write and debug the tests in Node.js so they
 // don't have to learn different tools and tech stacks just to create some UI tests.
-public class JavascriptTests : UITestBase
+public class JavaScriptTests : UITestBase
 {
-    public JavascriptTests(ITestOutputHelper testOutputHelper)
+    public JavaScriptTests(ITestOutputHelper testOutputHelper)
         : base(testOutputHelper)
     {
     }
 
     // Using this approach you only have to write minimal C# boilerplate, which you can see below.
     [Fact]
-    public Task ExampleJavascriptTestShouldWork() =>
+    public Task ExampleJavaScriptTestShouldWork() =>
         ExecuteTestAfterSetupAsync(context =>
         {
-            // Don't forget to mark the script files as "Copy if newer", so they are available to the test. It's best to
-            // include something like the following in your csproj file:
+            // Don't forget to mark the script files as "Copy if newer", so they are available to the test. If you
+            // include something like the following in your csproj file, then you only have to do this once:
             // <None Update="Tests\*.mjs" CopyToOutputDirectory="PreserveNewest" />
-            var workingDirectory = "Tests";
-            var scriptPath = Path.Join(workingDirectory, "test.mjs");
+            var scriptPath = Path.Join("Tests", "JavaScriptTests.mjs");
```

Comment:
Under https://github.com/Lombiq/UI-Testing-Toolbox/pull/421#discussion_r1846951255 I also suggested to recommend copying mjs files to the root of the output directory instead. That would make this easier simpler too, with just `"JavaScriptTests.mjs"` needed for the path.

Reply:
Fair enough.